client/src/components/admin/AdminManagePlans.jsx

```diff
@@ -14,7 +14,7 @@ const AdminManagePlans = () => {
   const allPlans = useSelector((state) => state.plans.allPlans);
   const updateLoader = useSelector((state) => state.plans.updateLoader);
   const loading = useSelector((state) => state.plans.loading);
-  console.log(loading, "state");
+  console.log(allPlans, "state");
```

Comment:
Console.log is not required.

Reply:
will do it. thanks!

Reply:
Why did you merge without resolving them?